Examples/Oscilloscopes/PerformanceScopes/src/FetchWaveformDataExample/README.md

```diff
@@ -0,0 +1,15 @@
+# Fetch Waveform Data Example
+Original attribution: Dave W. - Tektronix Applications
+
+This example demonstrates how to fetch waveform data from MSO/DSA/DPO  5K 7K & 70K Series Scopes, convert the raw data into floating point values and then save it to disk in a binary format.
+
+Compatible Instruments
+----------------------
+* MSO/DPO 5000(B) Series Oscilloscopes
+* DPO7000(C) Series Oscilloscopes
+* MSO/DSA/DPO 70000(B)(C)(DX)(SX) Series Oscilloscopes
+
```

Comment:
The markdown checker needs to be disabled for forum.tek links because they always timeout - just put `<!-- markdown-link-check-disable -->` here

Reply:
OK, I added the tag to the README.md file.  I think it should be fixed now.